### PR TITLE
Factor out smoketest setup into contextmanager

### PR DESCRIPTION
**Pull request by konradkonrad**
This encapsulates a large part of the smoketest setup into its own reusable contextmanager. This simplifies the code in `raiden/ui/cli.py` and allows progress on https://github.com/raiden-network/scenario-player/issues/491

### CODE REVIEW

**Comment by konradkonrad** [comment]
Don't review yet!

File: raiden/tests/utils/transport.py [hunk 1]
```diff
@@ -284,6 +286,7 @@ def make_requests_insecure():
     # Disable verification in requests by replacing the 'verify'
     # attribute with non-writable property that always returns `False`
     requests.Session.verify = property(lambda self: False, lambda self, val: None)  # type: ignore
+    urllib3.disable_warnings(InsecureRequestWarning)
```
Comment (konradkonrad) [inline raiden/tests/utils/transport.py]: From my point of view, calling `make_requests_insecure()` is bold enough, so we don't need to disable the warning explicitely.

File: raiden/tests/utils/eth_node.py [hunk 2]
```diff
@@ -202,13 +201,11 @@ def geth_keystore(datadir: str) -> str:
     return os.path.join(datadir, "keystore")
 
 
-def geth_keyfile(datadir: str, address: Address) -> str:
+def geth_keyfile(datadir: str) -> str:
```
Comment (hackaugusto) [inline raiden/tests/utils/eth_node.py]: can we make this a single function instead of two?

Comment (konradkonrad) [reply raiden/tests/utils/eth_node.py]: which two functions do you refer to? Is it
https://github.com/raiden-network/raiden/blob/961df1977fa93d267060b41c0041f831730552c8/raiden/tests/utils/eth_node.py#L204-L207

vs

https://github.com/raiden-network/raiden/blob/961df1977fa93d267060b41c0041f831730552c8/raiden/tests/utils/eth_node.py#L282-L285

?

Comment (hackaugusto) [reply raiden/tests/utils/eth_node.py]: I mean both of them, can we have a single function, say `eth_keyfile` that returns a path that can be used by either client?

**Pull request merged**